policyengine_us/tests/policy/baseline/gov/states/nc/tax/nc_use_tax.yaml

```diff
@@ -0,0 +1,39 @@
+- name: test 1
```

Comment:
Try and be more descriptive in the naming of your test cases
i.e NC use tax for agi of 3500